org.mapstruct.eclipse/plugin.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+     Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+     and/or other contributors as indicated by the @authors tag. See the
+     copyright.txt file in the distribution for a full listing of all
+     contributors.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         id="mapstructCompletionProposalComputer"
+         name="MapStruct Proposals"
+         point="org.eclipse.jdt.ui.javaCompletionProposalComputer">
```

Comment:
I didn't mean to interfere other content assists. But only show the MapStruct proposals "page" when auto-tabbing within a mapper class, within the `@Mapping` annotation.

E.g. say I have this piece of code:

    @Mapping(source = "id", target = "primaryKey")
     ^--- pos1         ^--- pos2

When auto-tabbing with the cursor at pos1, I get: "Default proposals" -> Ctrl+Space -> "Template Proposals" -> Ctrl+Space -> "SWT Template Proposals" -> Ctrl+Space -> "MapStruct proposals"

At pos2, I get "Java Proposals" -> Ctrl+Space -> "MapStruct proposals".

Ideally, I would not want to get the MapStruct proposals at pos1 (because there is nothing we have to offer), similar to as I don't get the "SWT Template Proposals" at pos2.

But it's no biggie, I'll open a separate issue for it.